.golangci.yml

```diff
@@ -0,0 +1,29 @@
+# This file configures github.com/golangci/golangci-lint.
+
+run:
+  timeout: 3m
+  tests: true
+  # skip auto-generated files.
+  skip-files:
```

Comment:
double cchecking if we want/is it already applying to abi bindings generated files

Reply:
Not at the golangci-lint level, but the CI job only lints the files in `teleporter/tests` by setting that as the target directory.

Reply:
@gwen917 Can you add `abi-bindings/go` to this list?

Reply:
For abi-bindings, some of the go files are auto generated. Do you want to do lint on autogened files? It may cause issue when we do run the abi checker CI. @cam-schultz